services/vision/client.go

```diff
@@ -242,6 +237,54 @@ func protoToObjects(pco []*commonpb.PointCloudObject) ([]*vision.Object, error)
 	return objects, nil
 }
 
+func (c *client) CaptureAllFromCamera(
+	ctx context.Context,
+	cameraName string,
+	returnImage bool,
+	returnDetections bool,
+	returnClassifications bool,
+	returnObject bool,
+	extra map[string]interface{},
+) (viscapture.VisCapture, error) {
+	ctx, span := trace.StartSpan(ctx, "service::vision::client::ClassificationsFromCamera")
+	defer span.End()
+	ext, err := protoutils.StructToStructPb(extra)
+	if err != nil {
+		return nil, err
+	}
+	resp, err := c.client.CaptureAllFromCamera(ctx, &pb.CaptureAllFromCameraRequest{
+		Name:                    c.name,
+		CameraName:              cameraName,
+		ReturnImage:             returnImage,
+		ReturnDetections:        returnDetections,
+		ReturnClassifications:   returnClassifications,
+		ReturnObjectPointClouds: returnObject,
+		Extra:                   ext,
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	dets, err := protoToDets(resp.Detections)
+	if err != nil {
+		return nil, err
+	}
+
+	objPCD, err := protoToObjects(resp.Objects)
+	if err != nil {
+		return nil, err
+	}
+
+	img, err := rimage.DecodeImage(ctx, resp.Image.Image, "")
+	if err != nil {
+		return nil, err
+	}
+
+	capt := viscapture.NewVisCapture(img, dets, protoToClas(resp.Classifications), objPCD)
```

Comment:
While `protoToClas` doesn't return an error, it was confusing at first because it seemed like you didn't return classifications. Go ahead and put this conversion on a line after `protosToDets` rather than doing the conversion in the  New function.